Sources/let-sExtend/SwiftUI/Styles/ColoredToggleStyle.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  File.swift
+//  
+//
+//  Created by ITgenerator on 07/01/2021.
+//
+
+import SwiftUI
+
+public struct ColoredToggleStyle: ToggleStyle {
+  
+  private let leftTintColor: Color
+  private let rightTintColor: Color
+  private let leftCircleColor: Color
+  private let rightCircleColor: Color
+  
+  public init(
+    leftTintColor: Color,
+    rightTintColor: Color,
+    leftCircleColor: Color,
+    rightCircleColor: Color
+  ) {
+    self.leftTintColor = leftTintColor
```

Comment:
maybe will be better to name in isOnTintColor, isOffTintColor, isOffCircleColor, issOnCircleColor